chest/core.py

```diff
@@ -134,8 +174,25 @@ def __init__(self, data=None, path=None, available_memory=None,
         self._on_miss = on_miss
         self._on_overflow = on_overflow
 
-    def __str__(self):
-        return '<chest at %s>' % self.path
+    def __repr__(self):
+        return '<chest at %s: keys=%s>' % (self.path, list(self._keys))
```

Comment:
This could be; however, it is closer to the repr for dict. I found it very hard to debug when I couldn't see what my keys were. I am happy to revert this.